tests/lib/internal-rules/internal-consistent-docs-description.js

```diff
@@ -0,0 +1,264 @@
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("internal-consistent-docs-description", rule, {
+    valid: [
+
+        // wrong exports format: "internal-no-valid-meta" reports this already
+        [
+            "module.exports = function(context) {",
+            "    return {",
+            "        Program: function(node) {}",
+            "    };",
+            "};"
+        ].join("\n"),
+
+        // missing `meta.docs.description` property: "internal-no-valid-meta" reports this already
+        [
+            "module.exports = {",
+            "    meta: {},",
+            "    create: function(context) {",
+            "        return {};",
+            "    }",
+            "};"
+        ].join("\n"),
+        [
+            "module.exports = {",
+            "    meta: {",
+            "        docs: {",
+            "            description: 'enforce some stuff'",
+            "        }",
+            "    },",
+
+            "    create: function(context) {",
+            "        return {};",
+            "    }",
+            "};"
+        ].join("\n"),
+        [
+            "module.exports = {",
+            "    meta: {",
+            "        docs: {",
+            "            description: 'require some things'",
+            "        }",
+            "    },",
+
+            "    create: function(context) {",
+            "        return {};",
+            "    }",
+            "};"
+        ].join("\n"),
+        [
+            "module.exports = {",
+            "    meta: {",
+            "        docs: {",
+            "            description: 'disallow bad things'",
+            "        }",
+            "    },",
+
+            "    create: function(context) {",
+            "        return {};",
+            "    }",
+            "};"
+        ].join("\n")
+    ],
+    invalid: [
+        {
+            code: [
+                "module.exports = {",
+                "    meta: {",
+                "        docs: {",
+                "            description: 'do stuff'",
+                "        }",
+                "    },",
+
+                "    create: function(context) {",
+                "        return {};",
+                "    }",
+                "};"
+            ].join("\n"),
+            errors: [{
+                message: "`meta.docs.description` should start with one of the following words: enforce, require, disallow. Started with \"do\" instead.",
+                line: 4,
+                column: 26
+            }]
+        },
+        {
+            code: [
+                "module.exports = {",
+                "    meta: {",
+                "        docs: {",
+                "            description: ''",
```

Comment:
I don't think this is a blocker, but something to consider - do we want to have an alternate error message if the description field is empty or is just whitespace? Might not be necessary since this is entirely internal and we'll know what this means :)